### PR TITLE
Expand cex_{chain}.flows to all EVM chains

### PR DESCRIPTION
**Pull request by hildobby**
Leveraging the cex_flows macro to expand the spells to all EVM chains.

~Waiting on https://github.com/duneanalytics/spellbook/pull/5588 before proceeding on this PR~ (Merged ☑️)

### CODE REVIEW

File: models/_sector/cex/flows/cex_celo_flows.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['flow_type', 'unique_key']
+)
+}}
+
+{{cex_flows(
+        blockchain = blockchain
+        , transfers = ref('tokens_scroll_transfers')
+        , addresses = ref('cex_scroll_addresses')
```
Comment (jeff-dude) [inline models/_sector/cex/flows/cex_celo_flows.sql:17]: still scroll here

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/_sector/cex/flows/cex_celo_flows.sql (hunk 1 above)
Comment (hildobby) [reply models/_sector/cex/flows/cex_celo_flows.sql]: changed to this in all `cex_{chain}.flows` spells for easier management:
```
, transfers = ref('tokens_' + blockchain + '_transfers')
, addresses = ref('cex_' + blockchain + '_addresses')
```

Comment (jeff-dude) [reply models/_sector/cex/flows/cex_celo_flows.sql]: love it!

**Review by jeff-dude — APPROVED** [review]

**Comment by jeff-dude** [comment]
successful CI run for reference:
https://github.com/duneanalytics/spellbook/actions/runs/8428509030/job/23081131336

**Pull request merged**